lib/mongoose_push/service/apns/state.ex

```diff
@@ -15,6 +15,15 @@ defmodule MongoosePush.Service.APNS.State do
     default_topics = setup_default_topics(pool_configs)
     :apns_state = :ets.new(:apns_state, [:set, :public, :named_table])
     true = :ets.insert(:apns_state, default_topics)
+
+    :telemetry.execute(
+      [:mongoose_push, :apns_state, :init],
```

Comment:
`init` events should always get their `terminate` counterpart in order to enable metrics like "number of active instances".